README.md

```diff
@@ -5,59 +5,53 @@
 ![GitHub License](https://img.shields.io/github/license/charmed-hpc/hpc-libs)
 [![Matrix](https://img.shields.io/matrix/ubuntu-hpc%3Amatrix.org?logo=matrix&label=ubuntu-hpc)](https://matrix.to/#/#hpc:ubuntu.com)
 
-A collection of charm libraries for authoring HPC charms 📖🖋️
+A collection of python libraries for authoring HPC charms 📖🖋️
 
-The hpc-libs charm libraries are used within the [Juju](https://juju.is) charms that compose
-Charmed HPC. They are standalone libraries, and should be managed
-as charm libraries, with installation via `charmcraft fetch-lib ...`, after which they may be
-imported and used as normal Python modules. The current charm libraries in hpc-libs include:
+The hpc-libs package is used within the [Juju](https://juju.is) charms that compose
+Charmed HPC. The current libraries shipped in hpc-libs include:
```

Comment:
[nit]: We could directly link to the charms architecture section of the Juju website. While you might need to do a little more exploring, the link is more succinct in the sense that it explains to reader what a "Juju charm" rather than just what Juju is: 

```suggestion
The hpc-libs package is used within the [Juju charms](https://juju.is/charms-architecture) that compose
Charmed HPC. The current libraries shipped in hpc-libs include:
```